jmix-pivottable/pivottable-flowui-kit/src/main/resources/META-INF/frontend/src/pivot-table/jmix-pivot-table.css

```diff
@@ -0,0 +1,398 @@
+jmix-pivot-table {
```

Comment:
<img width="452" alt="Screenshot 2024-10-02 at 11 43 06" src="https://github.com/user-attachments/assets/41854ebb-10e0-491f-8df7-59bf296da0a0">

Let's make header and body font-size the same